### ISSUE TITLE
Most links in Readme offline

### ISSUE THREAD

**Issue by myfairsyer** [adityaravishankar/command-and-conquer#81]
Besides the YouTube link, all links in the readme only show error messages:

| Link | URL |
| -- | -- |
| OSS demo | http://www.adityaravishankar.com/projects/games/command-and-conquer-demo/ |
| newer, proprietary demo | http://www.adityaravishankar.com/projects/games/command-and-conquer/ |
| blog post | http://www.adityaravishankar.com/2011/11/command-and-conquer-programming-an-rts-game-in-html5-and-javascript/ |
| facebook page | http://www.facebook.com/CommandConquerHtml5 |

**Comment by randomodbuild** [comment]
Yes, I've noticed this too. His website has been offline for a while. I've tried to send money to the paypal account on his original blog (went to archive.org) to solicit a response, to no success. I really miss playing this game and hope he's ok!

**Comment by quantuminformation** [comment]
dang
